src/plugins/embeddable/public/embeddable_panel/panel_actions/customize_panel_action/customize_panel_action.tsx

```diff
@@ -126,6 +127,7 @@ export class CustomizePanelAction implements Action<CustomizePanelActionContext>
       toMountPoint(
         <KibanaReactContextProvider>
           <CustomizePanelEditor
+            titleFocus={!trigger} // if no trigger, then execute was called from the title element being clicked
```

Comment:
nit: this seems like a pretty roundabout way to determine this. What about having a function called `openCustomizePanel` defined in the module that this `execute` method uses and the title click uses so it can skip the action entirely.

Reply:
Done in e9198dcbe833f3a518ef58c89f422f44e7197025 - it's a pretty significant code change, so if you could take another look, that would be awesome 🙇 Thank youuu